README.md

```diff
@@ -1 +1,188 @@
-# To-do-list-Testing
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="src/images/Cizzla_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>To-do List README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 To-Do list <a name="about-project"></a>
+
+
+
+**To-Do List** To-do list is a simple but functional app that helps the user to add and remove daily to do tasks.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp/">Javascript</a></li>
+  </ul>
+</details>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **Homepage**
+- **About**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li>Web Browser</li>
+    <li>Code Editor</li>
+    <li>Html Knowledge</li>
+    <li>Css Knowledge</li>
+    <li>Local file storage</li>
+</ul>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]( https://iankiema.github.io/Todo-list-webpack/)
```

Comment:
- [ ] It would be better if we fix the live demo link, currently the link is broken and returns page '404' error page not found